ddtrace/opentracer/tracer.py

```diff
@@ -6,6 +8,7 @@
 from ddtrace import Span as DatadogSpan
 from ddtrace import Tracer as DatadogTracer
 from ddtrace.constants import FILTERS_KEY
+from ddtrace.contrib.logging.patch import DDLogRecord
```

Comment:
```suggestion
from ddtrace.contrib.logging import DDLogRecord
```

We should expose `DDLogRecord` officially if it's going to be in the public API imo